resources/lib/objects/__init__.py

```diff
@@ -6,5 +6,6 @@
 from actions import Actions
 from actions import PlaylistWorker
 from actions import on_play, on_update, special_listener
+import utils
```

Comment:
It does, in fact, use all of the functions in utils.  Because there's exactly one function there, and as far as I can tell, it only gets called once in the entire codebase.  I'm playing with the idea of removing it altogether, but haven't gotten to it yet.

https://github.com/jellyfin/jellyfin-kodi/blob/5890814ae68629042615bbafcfb53968c606e95f/resources/lib/entrypoint/service.py#L79

TLDR: Yes, it could be more specific, but it's likely going away or getting moved in some fashion in the near future anyway